tools/ci_build/github/azure-pipelines/templates/win-ci.yml

```diff
@@ -262,6 +262,13 @@ jobs:
           msBuildCommandline: '"C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\MSBuild\Current\Bin\amd64\msbuild.exe" "$(Build.BinariesDirectory)\Debug\onnxruntime.sln" /p:platform="$(MsbuildPlatform)" /p:configuration=Debug /p:VisualStudioVersion="16.0" /m /p:PreferredToolArchitecture=x64'
           excludedPaths: '$(Build.BinariesDirectory)#$(Build.SourcesDirectory)\cmake#C:\program files (x86)'
 
+      - task: PostAnalysis@2
```

Comment:
This is the main change I want to add. All others are for making this task pass.